pajbot/modules/basic/permaban.py

```diff
@@ -78,6 +94,12 @@ def unpermaban_command(self, bot, source, message, **rest):
             if self.settings["unban_from_chat"] is True:
                 bot.unban(user)
 
+            if (
```

Comment:
I think this is a bit unintutive but there has to be a short delay (2 or 3 seconds) between unban and timeout, otherwise the timeout will not work, you can try it out yourself in normal chat. You could just do "bot.execute_later"

Reply:
Are you sure? This worked perfectly when I tested it in my own chat.

![image](https://user-images.githubusercontent.com/12804673/82731571-442b0700-9d4b-11ea-8428-6a0f879ca0cd.png)

Reply:
Oh wow, I made an assumption there, it does look like that's been fixed on Twitch's end. A delay used to be necessary